src/main/java/vic/cinema/Main.java

```diff
@@ -68,21 +70,21 @@ public static void main(String[] args) throws AuthenticationException {
 
         authenticationService.register(user.getEmail(), user.getPassword());
         User user2 = authenticationService.login(user.getEmail(), user.getPassword());
```

Comment:
let's wrap call to `login()` method in try-catch and add log.warn with the informative message in `catch` block

Reply:
Done